x/slashing/keeper/signing_info.go

```diff
@@ -209,3 +210,43 @@ func (k Keeper) GetValidatorMissedBlocks(ctx context.Context, addr sdk.ConsAddre
 
 	return missedBlocks, err
 }
+
```

Comment:
maybe add a doc:

// Updates cons address to its pub key relation
// Updates signing info (removes old one, and sets new one)
// Updates missed block address

Reply:
++